### ISSUE TITLE
integration tests failing

### ISSUE THREAD

**Issue by edublancas** [ploomber/jupysql#347]
I did a [changelog change](https://github.com/ploomber/jupysql/commit/ef95a0c4039e23e3f47affe274c0f4c916ba6d32) and the integration tests started failing

looks like a problem with psycopg2: https://github.com/ploomber/jupysql/actions/runs/4577876908/jobs/8083815193

cc @idomic @tonykploomber

**Comment by tonykploomber** [comment]
Yeah I am having this issue too...checking

**Comment by tonykploomber** [comment]
Something wrong with new version of pgspecial, they have a new release today
https://pypi.org/project/pgspecial/#history

PR is on the way

**Comment by edublancas** [comment]
pgspecial is an optional feature and most users don't have it anyway. the impact is near zero.

**Comment by edublancas** [comment]
No, those are two different issues. This issue is about pgspecial (optional dependency), you're taking about psycopg2 (I already reported that problem in https://github.com/ploomber/jupysql/issues/344)

**Comment by tonykploomber** [comment]
@edublancas 
They replied we should use psycopg3 instead of psycopg2
https://github.com/dbcli/pgspecial/issues/138#issuecomment-1493138055
> It seems that you are passing a psycopg 2 connection where a psycopg 3 one is expected. Maybe you upgraded the package, but not the dependencies.

Are you able to upgrade the psycopg3 package from your side?

**Comment by edublancas** [comment]
new issue to follow up: https://github.com/ploomber/jupysql/issues/367
